app/lib/widgets/wallets/add_edit_contact.dart

```diff
@@ -121,7 +126,7 @@ class _AddEditContactState extends State<AddEditContact> {
       return;
     }
     widget.onAddContact!(PkidContact(
-        name: contactName, address: contactAddress, type: widget.chainType));
+        name: contactName, address: contactAddress, type: chainType));

```

Comment:
This should be conditional on which contacts' type.
so if we are on stellar contacts, and adding tfchain contact shouldn't be added to the list